<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftBar</class>
 <widget class="QWidget" name="LeftBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>61</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeftBar</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btn_chat, #btn_contact, #btn_more,#btn_files,#btn_favourite,#btn_topStories,
#btn_miniProgram, #btn_phone
{
border:none;
};</string>
  </property>
  <widget class="QPushButton" name="btn_avatar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_chat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/chat_pressed.png</normaloff>:/resource/image/chat_pressed.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_contact">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/contact_normal.png</normaloff>:/resource/image/contact_normal.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_favourite">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/favourite_normal.png</normaloff>:/resource/image/favourite_normal.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_more">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/more.png</normaloff>:/resource/image/more.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_files">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/files.png</normaloff>:/resource/image/files.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_topStories">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/top_stories.png</normaloff>:/resource/image/top_stories.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_phone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>485</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/phone.png</normaloff>:/resource/image/phone.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_miniProgram">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/resource/image/mini_program.png</normaloff>:/resource/image/mini_program.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>Feedback</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionBackupAndRestore">
   <property name="text">
    <string>BackupAndRestore</string>
   </property>
   <property name="toolTip">
    <string>Backup and Restore</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
